### ISSUE TITLE
add pangu-onnx to pytorch

### ISSUE THREAD

**Issue by GotoHitori** [ENOT-AutoDL/onnx2torch#227]
Hi, I'd like to know if pangu-weather forecast model can be converted to pytoch, here is the link to the pangu-moedl: https://github.com/198808xc/Pangu-Weather

**Comment by dsuhoi** [comment]
@GotoHitori Yes, look at the [dialogue thread with the FuXi](https://github.com/ENOT-AutoDL/onnx2torch/issues/191) model (it has been verified that similar modifications of onnx2torch will solve your problem).
